routes/teams.js

```diff
@@ -40,9 +41,9 @@ var returnItem = function (item, res){
   res.json(item);
 }
 
-var createQuiz = function(rb_team_id, league, api_team_id, res, callback){
+var createQuiz = function(rb_team_id, league, api_team_id, team_name, res, callback){
```

Comment:
can you call this variable "quiz_name."If we do lists and what not, there will be no relevant "team_name"